data/nlp/twitter_preprocessing_recipe.py

```diff
@@ -0,0 +1,82 @@
+"""Preprocess the tweets by normalising username, removing unnecessatry punctuations, exapanding the hashtags"""
+import re
+import os
+import pandas as pd
+import datatable as dt
+from h2oaicore.systemutils import config
+from h2oaicore.data import CustomData
+
+text_colnames = ["text"]
+
+output_dataset_name = "df_preprocessed"
```

Comment:
cc @SudalaiRajkumar 
is there any recommendation here on the final outputname ?

Reply:
I can make it {original_file name}_processed

Reply:
@SudalaiRajkumar is there a way to do that?

Reply:
Am not sure whether there is a way to do that. Generally these names are edited by end user when they run the recipe. 

Thanks,